src/routes/lesson/lesson.ts

```diff
@@ -0,0 +1,47 @@
+import app, { Router, Request, Response } from "express";
+import lesson from "../../services/postgres/lesson";
+import responses from "../../utils/responses";
+
+const router = app.Router();
+
+interface lessonParam{
+    lesson_id: number;
+}
+
+router.get("/lessons", async (request, response) => {
+    await lesson.filterLessons(request.query).then(async function (result) {
+        return responses.simpleResponse(result, response);
+    });
+});
+
+router.get("/lesson/:lesson_id", async (request: Request<lessonParam>, response) => {
+    await lesson.getLessonById(request.params).then(async function (result) {
+        return responses.simpleResponse(result, response);
```

Comment:
Currently need to fix all of the lesson Param garbage (and other similar garbage) that was put here to complete the job